### ISSUE TITLE
Train details

### ISSUE THREAD

**Issue by pablodawson** [Xiaojiu-z/EasyControl#38]
Hey! thanks for sharing your code.

I created scripts for training with pytorch lightning, taking some scripts from OminiControl. 
Basically what I did:

- Set the attention processors to _MultiDoubleStreamBlockLoraProcessor/MultiSingleStreamBlockLoraProcessor_ on each transformer layer at init. Same settings as yours but without the weights.
- Set the trainable parameters to the "LoRALinearLayer" module parameters only.
- For each step VAE encode and patchify both the condition and input image, concatenate their ids. Make sure the condition is padded and of size 512x512.
- Prepare timestep, X_0, X_1, prompt embeds and guidance, same as Omini.
- Call the transformer with all of this, calculate flow matching loss. Empty the cache before doing so at every step.

Training with lr=1e-4, everything seems to be working as expected, but the results are very sub-par as the condition is ignored, and gradient checkpointing doesn't seem to work either. Only went up to a few 1000s steps, does it take longer to see results?
Am I missing something? Maybe I need to disable caching?

I understand you're releasing training scripts soon, if you prefer me to wait for it then no worries.

**Comment by pablodawson** [comment]
Letting it run it started to work
